environments/linux-amd64-wine_osxcross

```diff
@@ -74,11 +74,14 @@ RUN VERSION="6.2.1" ARCH="i386" gmp.sh
 
 RUN install-staging-libs.sh
 
-
 RUN VERSION="3.7.2" ARCH="x86_64" gnutls.sh
 RUN VERSION="3.7.2" ARCH="i386" gnutls.sh
 
+RUN VERSION="6.2" ARCH="x86_64" ncurses.sh
+RUN VERSION="6.2" ARCH="i386" ncurses.sh
 
+RUN VERSION="2.0.12" ARCH="x86_64" SDL2.sh
+RUN VERSION="2.0.12" ARCH="i386" SDL2.sh
```

Comment:
Bump this to the latest release then dump the now unneeded patch.

Reply:
Are you sure that the patch is now unneeded?

Reply:
Yep I'm sure, the problem was resolved upstream after my report.